fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -289,8 +291,13 @@ private boolean permissionContains(UserPermission.View permission,
     Function<Set<? extends Authorizable>, Boolean> containsAuth = resources ->
         resources
             .stream()
-            .anyMatch(view -> view.getName().equalsIgnoreCase(resourceName) &&
-                view.getAuthorizations().contains(authorization));
+            .anyMatch(view -> {
+              Set<Authorization> authorizations = Optional.ofNullable(
```

Comment:
The null was basically a guard for test cases, in reality it'll be deserialized to an empty `Set<Authorization>`. 

I wasn't planning on proactively pushing a new `fiat-api` out to clouddriver/front50/orca.